tests/test_bs.py

```diff
@@ -381,6 +382,25 @@ def save_log_on_exit(request):
         (out, err) = p.communicate()
 
 
+@pytest.fixture()
+def add_curl(request, setup_bootstrap):
```

Comment:
I don't get why bootstrap is required here?

Reply:
setup_bootstrap holds the deployment_id, this is the string common to all the deployed parts (bootstrap, client, poet, oracle etc.)

Reply:
yes but we don't use it in the code below. ( you check that pods are there but that's not needed ).  we only use the namespace from testconfig